src/Raven.Client/Documents/Session/DocumentSession.Patch.cs

```diff
@@ -99,12 +100,15 @@ public partial class DocumentSession
                 Values = { { "val", val } }
             }));
 
-            _patchInfo = new { DocId = key, Script = $"\"{script}\"", Path = path, Val = val };
+            _patchInfo.Add(new { DocId = key, Script = $"\"{script}\"", Path = path, Val = val });
         }
 
         public override string ToString()
         {
-            return _patchInfo.ToString();
+            var sb = new StringBuilder();
+            foreach (var v in _patchInfo)
```

Comment:
This overrides session.ToString().
Use session.Advanced.Defer and you will not need this at all.